packages/account/src/providers/fuel-core-schema.graphql

```diff
@@ -559,6 +570,10 @@ type Header {
   applicationHash: Bytes32!
 }
 
+enum HeaderVersion {
+  V1
+}
+
```

Comment:
@petertonysmith94 Just to confirm, you have updated this file by running the script `pnpm build:schema` from the `account/package.json` file, right?